Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM node:latest
 
-ADD . /app
+RUN mkdir -p /app
 WORKDIR /app
 
+COPY package.json /app/
 RUN npm install
+
+COPY . /app
+
+CMD [ "./zenbot.sh", "trade" ]
```

Comment:
I feel like it may confuse people, especially those who are new to Docker. Either way you should pass --paper by default so people don't accidentally trigger live trading